action.yml

```
@@ -37,35 +37,30 @@ runs:
           echo "ERROR: 'curl' is missing"
           exit 1
         fi
-    - id: get_devices_list
+    - id: create_deployment
       shell: bash
       run: |
         if [ ! -z ${{ inputs.mender_devices_group }} ]; then
-          MENDER_DEVICES_LIST=$(curl -s \
+          # Makes management 'create deployment for group' API call to Mender server, using curl
+          # https://docs.mender.io/api/#management-api-deployments-create-deployment-for-a-group-of-devices
+          RESPONSE=$(curl -s -X POST https://hosted.mender.io/api/management/v1/deployments/deployments/group/${{ inputs.mender_devices_group }} \
+            -H "Content-Type: application/json" \
+            -H "Accept: application/json" \
             -H "Authorization: Bearer ${{ inputs.mender_pat }}" \
-            ${{ inputs.mender_uri }}/api/management/v1/inventory/groups/${{ inputs.mender_devices_group }}/devices)
-          if [[ ${MENDER_DEVICES_LIST} != \[* ]]; then
-            echo "ERROR: failed to get devices list for group '${{ inputs.mender_devices_group }}', server: ${{ inputs.mender_uri }}"
-            echo "Server's response: ${MENDER_DEVICES_LIST}"
-            exit 1
-          fi
+            --data-raw "{\"name\": \"$mender_deployment_name\", \"artifact_name\": \"$mender_release_name\"}")
```

I believe you need to use the template input variables here.
```suggestion
            --data-raw "{\"name\": \"${{ inputs.mender_deployment_name }}\", \"artifact_name\": \"${{ inputs.mender_release_name }}\"}")
```

---

Thanks for catching ! I made the change